source/install/virtual/virtualbox.rst

```diff
@@ -2,20 +2,20 @@ Installing QIIME 2 using VirtualBox
 ===================================
 
 .. warning::
-   The following steps require approximately 15 GB of free disk space: 2.7 GB for the ZIP file, 3.3 GB for the unzipped ZIP file contents, and 10 GB for the installed Virtual Machine (VM). Once you have installed the VM you can free up 5 GB of space by deleting the ZIP + the unzipped ZIP file contents.
+   The following steps require at least approximately 25 GB of free disk space: ~5 GB for the ZIP file, ~5 GB for the unzipped ZIP file contents, and ~15 GB for the installed Virtual Machine (VM). Once you have installed the VM you can free up ~10 GB of space by deleting the ZIP + the unzipped ZIP file contents.
```

Comment:
"require at least approximately 25 GB" -> "require at least 25 GB"

Reply:
How about "require at least ~25 GB"?

Reply:
sure — "at least approximately" is just a bit of a mouthful...